### ISSUE TITLE
`flyctl secrets list --json` doesn't output JSON

### ISSUE THREAD

**Issue by abevoelker** [superfly/flyctl#1145]
**Describe the bug**
I wanted to get a list of environment variable names used by my app. The documentation for `flyctl secrets list` implies the global option `--json` should work for it, but it seems to only output table-style formatted text output. I tried with both `-j` and `--json` flags, and using the `-a` vs. `-c config` options but it made no difference.

* OS = Ubuntu 20.04
* 
	  $ fly version
	  flyctl v0.0.369 linux/amd64 Commit: 1df02229 BuildDate: 2022-08-09T10:27:58Z

** Paste your `fly.toml`

N/A

** Command output: **

```text
$ flyctl secrets list -j -a foo
NAME                            DIGEST                                  CREATED AT           
DATABASE_URL                    4cdb4e12946f0bfbba4651b703efded2        2022-08-05T20:18:48Z
BAR                             93bc63e0b4f48fbbff568d9fc0dc3def        2022-08-05T21:10:22Z
```

**Comment by jsierles** [comment]
Thanks - this will be released in the next version via #1146.

**Comment by abevoelker** [comment]
Thanks for the quick fix!
